packages/code-du-travail-frontend/src/common/BigLink.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "../../routes";
+import { getRouteBySource, getLabelBySource } from "../sources";
+import ArticleIcon from "../icons/ArticleIcon";
+import styled from "styled-components";
+
+export function BigLink({ icon, query, data }) {
+  const { slug, title, source, path = "" } = data._source;
```

Comment:
Why do you keep this `_source` elasticsearch notation ?
The frontend should be agnostic to the data source.

Reply:
totally agree but since it's return by the api everywhere I would prefer address the problem once than piece by piece. Receiving sometimes  _source + some ES info and sometimes not tend to be confusing, imho

Reply:
alright